packages/core/src/balance/Balance.chain.ts

```diff
@@ -40,7 +40,11 @@ export async function getBalance(
   accountAddress: IPublicIdentity['address']
 ): Promise<BN> {
   const blockchain = await BlockchainApiConnection.getConnectionOrConnect()
-  return (await blockchain.api.query.system.account(accountAddress)).data.free
```

Comment:
I wonder why this hasn't be an issue before as `...free` is of type `Codec`.

Reply:
This is probably there because you added „balance: u128“? That overwrites the internal balance type or something. Just a guess...

Reply:
I tested this thoroughly and I couldn't make it work without calling `toString()` or `toBn()` on this value, independent of setting `Balance: 'u128'` or `AccountInfo` in the custom types.

I would have loved to use `.toBn()` but for some reason the mock didn't know this function even after updating all Polkadot dependencies to the latest type, and setting the correct type for `system.accountInfo`.

Reply:
Ah ok. Was just an idea. 😄